website/docs/tooling/cli.md

```diff
@@ -81,8 +81,34 @@ function Comp(props) {
 
 ## Extraction
 
+<Tabs
+groupId="npm"
+defaultValue="npm"
+values={[
+{label: 'npm', value: 'npm'},
+{label: 'yarn', value: 'yarn'},
+]}>
+<TabItem value="npm">
+
+```sh
+npm run formatjs extract --help
```

Comment:
hmm I'm not sure this will work either. If we already declare custom scripts for `extract` & `compile` should this just be `npm run extract` or `yarn extract`?

Reply:
These commands do work as they are written, but that is definitely a thing which is fairly misleading about this doc page. It instructs you to set up the scripts and then forget about them :sweat_smile:

Reply:
the setup scripts should allow you to run `npm run extract` or `yarn extract` w/o the `formatjs` is what I meant

Reply:
Yep, I got that. I'll update the PR to change that as well.